docs/site/src/components/AppRouter.js

```diff
@@ -90,6 +90,13 @@ export default function AppRouter() {
             component={MarkdownDocs}
             nav
           />
+          <Route
```

Comment:
Why moving the section? I think that it was fine before.
It may be linked to what I said regarding it's important. I wasn't referring to that. I was saying that in the getting started section of the readme, I think that we should linked to how to setup the font.